ui/admin/app/routes/scopes/scope.js

```diff
@@ -29,14 +28,7 @@ export default class ScopesScopeRoute extends Route {
   async model({ scope_id: id }) {
     // Since only global and org scopes are authenticatable, we can infer type
     // from ID because global has a fixed ID.
-    const type = id === 'global' ? TYPE_SCOPE_GLOBAL : TYPE_SCOPE_ORG;
-    return this.store.findRecord('scope', id).catch(() => {
-      const maybeExistingScope = this.store.peekRecord('scope', id);
-      const scopeOptions = { id, type };
-      return (
-        maybeExistingScope || this.store.createRecord('scope', scopeOptions)
-      );
-    });
+    return this.store.findRecord('scope', id);
```

Comment:
None blocking comment: The previous implementation returned a "maybeExistingScope" or it creates a new record for scope. I do understand this changes validates/assumes the scope record its ALWAYS present?